swift-sdk/Internal/InternalIterableAPI.swift

```diff
@@ -168,18 +169,12 @@ final class InternalIterableAPI: NSObject, PushTrackerProtocol, AuthProvider {
     }
     
     func getMergeDefaultValue(merge: Bool?) -> Bool {
-        let anonUser = localStorage.userIdAnnon;
-        let isEmailOrUserId = isEitherUserIdOrEmailSet();
+        //let anonUser = localStorage.userIdAnnon;
+        //let isEmailOrUserId = isEitherUserIdOrEmailSet();
         if let merge = merge {
-               return merge
-           } else {
-               if (!isEmailOrUserId && anonUser == nil) { // Criteria is not yet met (default merge is true)
-                   return true;
-            } else if (!isEmailOrUserId && anonUser != nil) { // Criteria is met (Iterable profile created with an autogenerated identity)(default merge is true)
-                return true;
-            } else { // Current logged in user is identified (default merge is false)
-                return false;
-            }
+            return merge
+        } else {
+            return true
```

Comment:
Why is this function required anymore if an `if` if the only remaining logic?

Reply:
Yeah true we can simplify and just add a conditional to the function body